solr/test-framework/src/java/org/apache/solr/cloud/AbstractUnloadDistributedZkTestBase.java

```diff
@@ -285,7 +287,11 @@ private void testCoreUnloadAndLeaders() throws Exception {
     // collectionClient.commit();
 
     // unload the leader
-    try (SolrClient collectionClient = getHttpSolrClient(leaderProps.getBaseUrl(), 15000, 30000)) {
+    try (SolrClient collectionClient =
+        new HttpSolrClient.Builder(leaderProps.getBaseUrl())
+            .withConnectionTimeout(15000, TimeUnit.MILLISECONDS)
+            .withSocketTimeout(30000, TimeUnit.MILLISECONDS)
+            .build()) {
```

Comment:
This test class seems to always want a client with specific timeouts.  Then create a utility method _in this test_ for it.